qiling/core.py

```diff
@@ -238,7 +238,7 @@ def enable_lib_patch(self):
 
 
     # save all qiling instance states
-    def save(self, reg=True, mem=True, fd=False, cpu_context=False, snapshot=None):
+    def save(self, reg=True, mem=True, fd=False, cpu_context=False, os_context=True, loader=True, snapshot=None):
```

Comment:
should os_context and loader = False ?

Reply:
Whatever you think will serve the project better.